packages/node_modules/@ciscospark/plugin-phone/src/phone.js

```diff
@@ -174,6 +174,31 @@ const Phone = SparkPlugin.extend({
     return WebRTCMediaEngine.getUserMedia(constraints);
   },
 
+  /**
+   * Fetches a list of all of the current user's active calls
+   * @instance
+   * @memberOf Phone
+   * @returns {Promise<Call[]>}
+   */
+  listActiveCalls() {
+    return this.spark.internal.locus.list()
+      .then((loci) => {
+        const calls = [];
+        if (!loci) {
+          return calls;
+        }
+        loci.forEach((locus) => {
```

Comment:
Instead of creating a call for each locus and then checking if it's been emitted, can we check if it's been emitted and, if not, create it? then once you've iterated over the retrieved loci, you can return something like `Array.from(this.emittedCalls)`. Part of the goal here is to ensure only one call is ever created for a given audio-video experience.

Reply:
oh, i guess `Array.from` wouldn't work. I was thinking `emittedCalls` was a `Set`, but something like `clone(this.emittedCalls.models)`, maybe.